search/docker-compose.yml

```diff
@@ -82,17 +82,21 @@ services:
   reactivesearch:
     image: my-reactivesearch
     container_name: reactivesearch
-    build: .
-    # volumes:
-    #   - ./:/code # for development
-    #   - /code/node_modules # an exception from the above volume
+    build:
+      context: .
+      dockerfile:  ${reactivesearch_dockerfile:-Dockerfile}
+    volumes:
+       - ./:/code # for development, unused in production
```

Comment:
at the moment I have no way to switch this off in prod.

One solution might be using [jinja-compose](https://github.com/sinzlab/jinja-compose) or another way of templating the docker-compose file.

Reply:
if we keep the code repo under /usr/share/, then everything might be mountable

Reply:
I think for this one the easiest way is to have 2 docker-compose.yml files for dev and prod separately.